Extensions/probing.hh

```diff
@@ -329,14 +334,14 @@ inline double getReactivityScore(const Subsequence &leftBase, const bool isUnpai
 			}
 		}
 		if ((strcmp(getProbing_normalization(), "asProbabilities") == 0)) {
-			double max = 0;
+			double max = 0.0;
 			for(std::vector<double>::iterator it = probingData.begin(); it != probingData.end(); it++) {
 				if (max < *it) max = *it;
-				if (*it < 0) *it = 0;
+				if (*it < 0.0) *it = 0.0;
 			}
-			if (max > 0) {
+			if (max > 0.0) {
 				for(std::vector<double>::iterator it = probingData.begin(); it != probingData.end(); it++) {
-					*it = ((int) ((*it / max) * 10)) / 10.0;
+					*it = ((double) ((*it / max) * 10.0)) / 10.0;
```

Comment:
you are changing the data type from int to double. I don't recall why I used int here (for efficiency? for "precision"? for rounding?). I tried to by in sync with what RNAstructure https://rna.urmc.rochester.edu/RNAstructure.html is doing, to ease comparison against their implementation (they were the first to add in probing data). 

Do you have specific reasons for introducing this change? Did RNAstructure maybe also changed that in the meantime?

Reply:
> Did RNAstructure maybe also changed that in the meantime?
Not that I know of. 
I changed it here to get more precise results. But if the idea is to be synced with RNAstructure I'll change it back to int.

Reply:
hm ... we should also think about the use in pareto fronts. For that gapc offers the flag `--float-accuracy` - which will only work if the result of the left algebra is of type float (not int). Therefore, it could be work keeping it as float - or is that handled in the algebra itself?

Reply:
The return variable (score) of the function is a double, and therefore the choice function also works on doubles. So that should be taken care of already.
Or is it important that it really is of tpye float and not double? 

We should change that for the mfe algebra as well because there the answer is of type int.

Reply:
> We should change that for the mfe algebra as well because there the answer is of type int.

please don't! That would make mfe computation much slower, since int is way cheaper than double.

Reply:
> Or is it important that it really is of tpye float and not double?

sorry, coding python right now I confused double with float

Reply:
But will the `--floag-accuracy` flag still work with the mfe values being integers? Or do we have to cast it a double at a later point?

Reply:
that is a good point, but it depends on the algebra product: for `mfe ^ probing` the `--float-accuracy` won't have an effect since precision of `mfe` cannot be changed. However, if we operate on `probing ^ mfe` it will work, i.e. algebra products for the pareto product operator `^` are not symmetric!